FLEDGE.md

```diff
@@ -184,6 +185,7 @@ The function gets called once for each candidate ad in the auction.  The argumen
 *   ad\_metadata: Arbitrary metadata provided by the buyer
 *   bid: A numerical bid value
 *   auction\_config: the auction configuration object passed to navigator.runAdAuction()
+*   trusted\_scoring\_signals: A value retrieved from a real-time trusted server chosen by the seller and reflecting the seller's opinion of this particular creative, as further described in [3.1 Fetching Real-Time Data from a Trusted Server](#31-fetching-real-time-data-from-a-trusted-server) below.
```

Comment:
It may be worthwhile to mention what happens in the case of [ads composed of multiple pieces](https://github.com/WICG/turtledove/blob/main/FLEDGE.md#34-ads-composed-of-multiple-pieces); alternatively, clarification of the product-level ad behavior for `score_ad` can be done in a separate change.